server/utils/constants.ts

```diff
@@ -16,6 +16,8 @@
 export const BASE_API_PATH = '/api/reporting';
 export const NODE_API = Object.freeze({
   GENERATE_REPORT: `${BASE_API_PATH}/generateReport`,
+  GET_REPORTS: `${BASE_API_PATH}/reports`,
+  GET_REPORT: `${BASE_API_PATH}/reports/{reportId}`,
```

Comment:
How do you feel about changing this to something like `GET_SINGLE_REPORT`? `GET_REPORT` vs `GET_REPORTS` seems too similar to me

Reply:
How does `GET_REPORT_BY_ID` sound to you?

Reply:
👍